### PR TITLE
#58 add a test for indexes, to check the relation to properties

### PR DESCRIPTION
**Pull request by romaintb**
PR about #58 

I am not sure you want this test in this file, hope this is okay though.

### CODE REVIEW

File: test/integration-orientdb/tests/define/indexes.js [hunk 1]
```diff
@@ -39,5 +40,18 @@ describe('Define related Operations', function() {
       });
     });
 
+    it('should properly create Link property from collection', function(done) {
```
Comment (romaintb) [inline test/integration-orientdb/tests/define/indexes.js]: the name of the test is a bad copy/paste ... d'oh

Comment (romaintb) [inline test/integration-orientdb/tests/define/indexes.js]: would this seem ok to you ?

```diff
-    it('should properly create Link property from collection', function(done) {
+    it('should properly create Link property for props', function(done) {
```

Comment (dmarcelino) [inline test/integration-orientdb/tests/define/indexes.js]: How about 'should properly create Link property in one-to-many association', just so we know/remember the specific objective of this test.

Comment (romaintb) [inline test/integration-orientdb/tests/define/indexes.js]: sure, will update the PR

**Comment by romaintb** [comment]
updated title of the test

**Pull request merged**